keps/sig-node/688-pod-overhead/README.md

```diff
@@ -311,12 +318,114 @@ Cons:
  * Not user perceptible from a workload perspective.
  * very complicated if the runtimeClass policy changes after workloads are running
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+Skipping this section as the feature was already rolled out to all supported k8s versions.
+
+### Monitoring Requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Using metrics mentioned in documentation https://kubernetes.io/docs/concepts/scheduling-eviction/pod-overhead/#observability:
+
+- `kube_pod_overhead_cpu_cores`
+- `kube_pod_overhead_memory_bytes`
+
+###### How can someone using this feature know that it is working for their instance?
+
+Using metrics mentioned in documentation https://kubernetes.io/docs/concepts/scheduling-eviction/pod-overhead/#observability:
+
+- `kube_pod_overhead_cpu_cores`
+- `kube_pod_overhead_memory_bytes`
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+N/A
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+N/A
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+No
+
+### Dependencies
+
+No
+
+###### Does this feature depend on any specific services running in the cluster?
+
+No
+
+### Scalability
+
+
+###### Will enabling / using this feature result in any new API calls?
+
+No
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+No
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+No
```

Comment:
Could the need to update the Overhead result in slower Pod operations?

Reply:
it is just an `if` and assignment since the runtimeclass object is already pulled: https://github.com/kubernetes/kubernetes/blob/3e9eddb8fe8733a4ac73ef070012376a045c90cf/plugin/pkg/admission/runtimeclass/admission.go#L119-L122 So unlikely.